spec/asyncapi.md

```diff
@@ -4,7 +4,7 @@
 
 Part of this content has been taken from the great work done by the folks at the [OpenAPI Initiative](https://openapis.org). Mainly because **it's a great work** and we want to keep as much compatibility as possible with the [OpenAPI Specification](https://github.com/OAI/OpenAPI-Specification).
 
-#### Version 2.0.0
+#### Version 2.0.1
```

Comment:
no need to change a version, you are just adding/fixing/improving the explanation, nothing changes in the spec itself.